packages/atomic/src/components/atomic-result-children/atomic-result-children.tsx

```diff
@@ -26,15 +34,23 @@ export class AtomicResultChildren {
   @State() public ready = false;
   @State() public templateHasError = false;
 
+  @Prop() inheritTemplates = false;
```

Comment:
add documentation for this prop in this PR

Reply:
How about we make this true by default ? 🤔

What would be the cons ?

Reply:
what I can think of is: the top level one (that can't really inherit the templates) will have that setting by default, so will never get this warning
```js
    if (!childrenTemplates.length && !this.inheritTemplates) {
      this.error = new Error(
        `The "${componentTag}" component requires at least one "${childTemplateComponent}" component.`
      );
      return;
    }
```
unless we do some sort of extra checking later on when all events have been fired, for example in the render method, but I'm assuming that's not ideal... or maybe in `ChildTemplatesContext` we can send the config, like `ChildTemplatesContext(inheritsTemplate)` and if it does but there is nothing to inherit, like 
```ts
      const event = buildCustomEvent(
        childTemplatesContextEventName,
        (result: ResultTemplatesManager<TemplateContent> | undefined) => {
          const resultListManager = (this as AtomicResultChildren)
            .resultListCommon?.resultTemplatesManager !== result ? result : null;
          if (inheritTemplate && !resultListManager) {
            this.error = new Error(
             `The "${componentTag}" component requires at least one "${childTemplateComponent}" component.`
            );
          } else {
             this[resultVariable] = resultListManager;
          }
        }
      );
```
I'll do that and see how it looks!

Reply:
actually, this doesn't really work.. we can't validate this unless we look at the parents..
I think leaving it as false makes the behaviour a bit more explicit, and makes it so that we don't need to distinguish between the child and the grandchild in the code for the validation (the user could still set `inherit-templates=true` for the top level children and not get the warning, but at least they will have read the docs about this feature and that should tell them that it only works for grandchildren.